app/models/user.rb

```diff
@@ -9,6 +24,15 @@ class User < ActiveRecord::Base
   has_and_belongs_to_many :publications
   has_and_belongs_to_many :projects
 
+  validates :personnel_number, numericality: { only_integer: true }
+  validate :personnel_number_in_range
+
+  def personnel_number_in_range
```

Comment:
Dafür kannst du eine Validation benutzen:
validates_inclusion_of :personnel_number, :in => 0..999999999